modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBAtlasLocalContainer.java

```diff
@@ -0,0 +1,43 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+public class MongoDBAtlasLocalContainer extends GenericContainer<MongoDBAtlasLocalContainer> {
```

Comment:
wonder if can be renamed to `MongoDbAtlasContainer` instead. WDYT?

Reply:
My feeling is Atlas Local is the more consistent name with MongoDB's own documentation and naming conventions:
https://hub.docker.com/r/mongodb/mongodb-atlas-local
https://www.mongodb.com/docs/atlas/cli/current/atlas-cli-deploy-docker/